tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -1005,7 +1010,9 @@ def test_grid_sync_rawkernel(self):
             x1 = cupy.arange(n ** 2, dtype='float32').reshape(n, n)
             x2 = cupy.ones((n, n), dtype='float32')
             y = cupy.zeros((n, n), dtype='float32')
-            kern_grid_sync((n,), (n,), (x1, x2, y, n ** 2))
+            block = self.block
+            grid = (n * n + block - 1) // block
+            kern_grid_sync((grid,), (block,), (x1, x2, y, n ** 2))
```

Comment:
So you actually caught another bug we encountered earlier! I wonder why the driver API would not fail with excessive arguments...

Reply:
That certainly made me wonder too..